lib/web/apiserver.go

```diff
@@ -893,6 +898,10 @@ func (h *Handler) bindDefaultEndpoints() {
 	// Returns logins included in the Connect My Computer role of the user.
 	// Returns an empty list of logins if the user does not have a Connect My Computer role assigned.
 	h.GET("/webapi/connectmycomputer/logins", h.WithAuth(h.connectMyComputerLoginsList))
+
+	// Implements the agent version server.
+	// Channel can contain "/", hence the use of a catch-all parameter
+	h.GET("/webapi/automaticupgrades/*request", h.WithUnauthenticatedHighLimiter(h.automaticUpgrades))
```

Comment:
Would it make sense to include "/channel" in the path? `/webapi/automaticupgrades/channel/*request`? Just so it's a bit more future-proof in case we want to add another API under /webapi/automaticupgrades in future.